index.html

```diff
@@ -18,34 +21,46 @@
             </nav>
     </header>
     <main>
+        <div class="header-desktop">
+            <div class="nav-desktop">
+                <ul>
+                    <li><a href="#">Portfolio</a></li>
+                    <li><a href="#">About</a></li>
+                    <li><a href="#">Contact</a></li>
+                </ul>
+            </div>
+        </div>
         <section class="heading-text headline">
-            <h1 class="primary">
-                <span class="color1">
-                    Hey there. <br>
-                    I'm Burhan. <br>
-                </span>
-                <span class="color2">I'm a software <br> developer</span>
-            </h1>
-            <p class="supporting-text">
-                I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, 
-                don't hesitate to contact me.
-            </p>        
+            <div class="text-container">
+                <h1 class="primary">
+                    <span class="color1 intro">
+                        Hey there. I'm Burhan. <br>
+                    </span>
+                    <span class="color2 intro">I'm a software developer</span>
+                </h1>
+                <p class="supporting-text">
+                    I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don't hesitate to contact me.
+                </p>
+            </div>    
             <div class="icon-container">
                 <ul class="icons">
                     <li><a href="https://twitter.com/@BurhanU14173360"><img src="assets/twitter.svg" alt="Twitter"></a></li>
                     <li><a href="https://www.linkedin.com/in/bhobserver"><img src="assets/linkedin.svg" alt="LinkedIn"></a></li>
                     <li><a href="#"><img src="assets/medium.svg" alt="medium"></a></li>
-                    <li><a href="https://github.com/bhobserver"><img src="Assets/github.svg" alt="GitHub"></a></li> 
+                    <li><a href="https://github.com/bhobserver"><img src="assets/github.svg" alt="GitHub"></a></li> 
                     <li><a href="#"><img src="assets/angellist.svg" alt="Angellist"></a></li>
                </ul>
             </div>      
+            <div class="bottom-arrow">
+                <img src="assets/bottom arrow.svg" alt="bottom-arrow" >
+            </div>   
```

Comment:
- [ ] As requested by the previous reviewer kindly consider sticking to the Figma design as much as possible  by decreasing the space between the background image and social media icons to make your design an exact duplicate of the given one  check the images below to see the difference 
- your design 👇 
<img width="1399" alt="Screen Shot 2023-06-17 at 23 03 27" src="https://github.com/BHObserver/portfolio/assets/60146030/5aaf35d4-800e-402e-b09f-a3d11d646344">


- Figma design 👇 
<img width="788" alt="Screen Shot 2023-06-17 at 23 03 58" src="https://github.com/BHObserver/portfolio/assets/60146030/5ebc0297-1c65-4386-b951-3aefea506f79">